app/models/remote_execution_provider.rb

```diff
@@ -122,6 +122,10 @@ def proxy_action_class
       'Proxy::RemoteExecution::Ssh::Actions::RunScript'
     end
 
+    def proxy_batch_size
+      Setting['foreman_tasks_proxy_batch_size']
```

Comment:
Would it make sense to have yet another setting for this?

Reply:
I don't believe so, I think this one make sense to share for now :thinking: